trogdor/README.md

```diff
@@ -10,54 +10,62 @@ First, we want to start a single-node Kafka cluster with a ZooKeeper and a broke
 
 Running ZooKeeper:
 
-    > ./bin/zookeeper-server-start.sh ./config/zookeeper.properties &> /tmp/zookeeper.log &
-
+```
+ ./bin/zookeeper-server-start.sh ./config/zookeeper.properties &> /tmp/zookeeper.log &
+```
 Running Kafka:
 
-    > ./bin/kafka-server-start.sh ./config/server.properties &> /tmp/kafka.log &
-
+```
+./bin/kafka-server-start.sh ./config/server.properties &> /tmp/kafka.log &
+```
 Then, we want to run a Trogdor Agent, plus a Trogdor Coordinator.
 
 To run the Trogdor Agent:
 
-    > ./bin/trogdor.sh agent -c ./config/trogdor.conf -n node0 &> /tmp/trogdor-agent.log &
-
+```
+./bin/trogdor.sh agent -c ./config/trogdor.conf -n node0 &> /tmp/trogdor-agent.log &
+```
 To run the Trogdor Coordinator:
 
-    > ./bin/trogdor.sh coordinator -c ./config/trogdor.conf -n node0 &> /tmp/trogdor-coordinator.log &
-
+```
+./bin/trogdor.sh coordinator -c ./config/trogdor.conf -n node0 &> /tmp/trogdor-coordinator.log &
+```
 Let's confirm that all of the daemons are running:
 
-    > jps
-    116212 Coordinator
-    115188 QuorumPeerMain
-    116571 Jps
-    115420 Kafka
-    115694 Agent
-
+```
+jps
```

Comment:
nit: for the commands that have output, losing the > means that it's not as clear what is the command and what is the output. (The new line might be clear enough)

Seems like an ok enough tradeoff to ensure that folks don't overwrite the files though.

Reply:
And given this was already the case for copy pasting the whole thing, I'd say this is a net positive.